README.md

```diff
@@ -5,9 +5,9 @@ A great example repo you can fork to your heart’s content, that serves as a la
 
 ## Getting started
 
-The first thing you want to do with this repo is quite likely **fork it**.  Use the Fork button in the top-right section of the page to do this.
+The first thing you want to do with this repo is quite likely [**fork it**](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo).  Use the Fork button in the top-right section of the page to do this.
 
-Once you’re on your fork, you can experiment however you like with it, play with branches, issues, milestones, labels, the wiki, and more.
+Once you’re on your fork, you can experiment however you like with it, play with [branches](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-branches), [issues](https://guides.github.com/features/issues/), [milestones], [labels], [the wiki], and more.
```

Comment:
The link for milestone is here: 
https://docs.github.com/en/github/managing-your-work-on-github/about-milestones